src/styles/blocks/body.scss

```diff
@@ -0,0 +1,10 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: normal;
+  font-style: normal;
+}
+
+.body {
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the block with the class `.body` is not necessary since you can apply these styles directly to the `body` tag. It's a common practice to set the margin and padding to 0 on the `body` tag to remove the default browser styling.